README.md

```diff
@@ -170,7 +171,7 @@ make up
 
 ### Multi-tenancy
 
-Multiple users on the hosts may create their own instances of Usernetes, but the port numbers have to be changed to avoid conflicts.
+Multiple users on the hosts may create their own instances of Usernetes. For systems that do not allow the lower port range, or for multiple usernetes deployments on the same physical node (experimental), the port numbers can be changed.
```

Comment:
"multiple usernetes deployments on the same physical node" is different from "Multiple users on the hosts" ?

Reply:
Yes. You could have a single user that has a physical node under a job, and on that node create two usernetes "nodes." That is different from two users having jobs on the same node, and both wanting their own usernetes node. Both cases need to consider conflict of ports. For the second reason, the customization is needed for centers that are strict about users only having access to higher ranges.

Reply:
Sounds too complicated to mix up multiple topics in this "Multi-tenancy" section here.